src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String VALUE_SEPARATOR = ",";
+    private static final String SUPPLY_VALUE = "supply";
+    private static final String BUY_VALUE = "buy";
+    private static final String RESULT_VALUE = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, reportFromData(readFromFile(fromFileName)));
```

Comment:
Better to introduce local variables, next time don't forget about that rule - keep your code clean and readable

Reply:
Thank you for your comments )